wellcomecollection/developers.wellcomecollection.org
# Upgrade every package

## What does this change?

Everything but also seemingly nothing.
Noticed many [security flags](https://github.com/wellcomecollection/developers.wellcomecollection.org/security/dependabot?q=is%3Aopen+manifest%3Ayarn.lock+package%3Apath-to-regexp) and Dependabot PRs
I think no one actively looks after this so it makes sense, and as I'm in between tickets I went for a wild mass upgrade. 
Not much needed updating, I'll comment in-line.

I've also added a CODEOWNERS file for future PRs to get picked up.

## How to test

Run it locally (`yarn && yarn start`) and build (`yarn build`); is it happy?
But I'm not versed in this repo at all so will take any extra testing you might think about! Like I haven't run `swizzle` and `clear` (see `package.json`), but they're not in the README file and therefore am thinking might not be ever used?

## How can we measure success?

No security flags woo.

## Have we considered potential risks?
 Breaks the site and we fix it.
